api/src/main/resources/workbench-api.yaml

```diff
@@ -7942,69 +7956,204 @@ definitions:
         description: Time at which snapshot is retrieved. Epoch millis.
         type: integer
         format: int64
-      researchers:
+      users:
         type: array
         items:
-          "$ref": "#/definitions/ReportingResearcher"
+          "$ref": "#/definitions/BqDtoUser"
       workspaces:
         type: array
         items:
-          "$ref": "#/definitions/ReportingWorkspace"
-  ReportingWorkspace:
-    description: >
+          "$ref": "#/definitions/BqDtoWorkspace"
+  BqDtoUser:
     type: object
     properties:
-      workspaceId:
-        description: PK for workspace table.
-        type: integer
-        format: int64
-      creatorId:
-        description: FK researcher ID for creator of workspace.
+      aboutYou:
+        type: string
+      areaOfResearch:
+        type: string
+      betaAccessBypassTime:
```

Comment:
The following fields are obsolete and I would suggest simply omitting them from the first cut:

- betaAccess*
- emailVerification*
- idVerification*
- phoneNumber
- organization

Another field I'd prefer not to export because of ambiguous semantics (technically, we don't have a concept of eRA commons expiration today):
- eraCommonsLinkExpireTime

Reply:
I'm assuming these changes should be integrated before this PR is merged. Please let me know if you're planning otherwise.